infra/mock_api/template/rds.tf

```
@@ -1,6 +1,40 @@
-# rds config goes here
-# need db user and pw
-# need security group
-# use postgres instance
+data "aws_ssm_parameter" "db_username" {
+  name = "POSTGRES_USER"
+}
+data "aws_ssm_parameter" "db_pw" {
+  name = "POSTGRES_PASSWORD"
+}
+data "aws_ssm_parameter" "db" {
+  name = "POSTGRES_DB"
+}
+data "aws_ssm_parameter" "db_host" {
+  name = "DB_HOST"
+}
+resource "aws_security_group" "rds" {
+  description = "allows connections to RDS"
+  name        = "rds-instance"
```

```suggestion
  name        = "mock-api-${var.environment_name}-rds-instance"
```

(something like that, to ensure it's environment-specific)